### PR TITLE
route53_info: Add RETURN block

### PR DESCRIPTION
**Pull request by mandar242**
##### SUMMARY

Currently route53_info is mising a [return block](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block).

This is a follow up on #1236 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

route53_info

### CODE REVIEW

**Review by jatorcasso — COMMENTED** [review]
can you please add `version_added` since these are new return values. Also, is it worth documenting the deprecated return values? Im not sure if thats necessary or not

**Comment by mandar242** [comment]
> can you please add `version_added` since these are new return values. Also, is it worth documenting the deprecated return values? Im not sure if thats necessary or not

sure added `version_added`. I'm not sure which values you mean here 'documenting the deprecated return values' ?

**Comment by jatorcasso** [comment]
> > can you please add `version_added` since these are new return values. Also, is it worth documenting the deprecated return values? Im not sure if thats necessary or not
> 
> 
> 
> sure added `version_added`. I'm not sure which values you mean here 'documenting the deprecated return values' ?

The CamelCased keys that will still be returned until we remove them

**Review by jatorcasso — APPROVED** [review]
I think version added should be 3.4.0 if I'm not mistaken - aside from that LGTM!

File: plugins/modules/route53_info.py [hunk 1]
```diff
@@ -204,6 +204,185 @@
...
+                    sample: 'test-string-to-match'
+                type:
+                    description: The type of the health check.
+                    type: str
+                    sample: HTTPS
+    version_added: 4.0.0
+ResourceRecordSets:
+    description: A deprecated CamelCased list of resource record sets returned by list_resource_record_sets in boto3. \
+                 This list contains same elements/parameters as it's snake_cased version mentioned above. \
+                 This field is deprecated and will be removed in next major version release.
```
[the remark this answers is not in the document]
Comment (mandar242) [reply plugins/modules/route53_info.py]: Added.

**Review by gravesm — APPROVED** [review]

**Pull request merged**